### DISCUSSION TITLE
'Invalid Argument Name' while creating stored procedure/function

### DISCUSSION THREAD

**Discussion by cem-al** [supabase/supabase#4038] [Questions]
Hi 👋

I'm trying to create a Stored Procedure using the 'Functions' tool under 'Database'.

However, regardless of the argument name & type it always gives the error 'Invalid Argument Name'

I'm quite new to development and might be missing something obvious...any help would be greatly appreciated 😊

```SQL
SELECT * FROM reviews 
INNER JOIN "ARTIST" ON reviews.artist_id = "ARTIST".id 
INNER JOIN "ALBUMS" ON reviews.album_id = "ALBUMS".id 
WHERE reviews.artist_id=(SELECT id FROM "ARTIST" WHERE artist_name = artist_lookup) 
AND reviews.album_id=(SELECT id FROM "ALBUMS" WHERE album_name= album_lookup)
```
![error](https://user-images.githubusercontent.com/60609268/143295688-caa1e80c-3e50-4ec1-b10f-ec05b3eb2cc3.PNG)

**Comment by GaryAustin1** [comment]
I'm thinking the UI has recently been broken.  I see the same thing.
